qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/MainActivity.kt

```diff
@@ -62,74 +52,49 @@ class MainActivity : AppCompatActivity(), PermissionsListener {
         )
     }
 
-    override fun onExplanationNeeded(permissionsToExplain: List<String>) {
-        Toast
-            .makeText(
-                this,
-                "This app needs location and storage permissions" +
-                    "in order to show its functionality.",
-                Toast.LENGTH_LONG
-            ).show()
-    }
-
-    override fun onPermissionResult(granted: Boolean) {
-        if (granted) {
-            requestPermissionIfNotGranted(Manifest.permission.WRITE_EXTERNAL_STORAGE)
-        } else {
-            Toast.makeText(this, "You didn't grant location permissions.", Toast.LENGTH_LONG).show()
```

Comment:
I removed this but probably need to bring it back. Can do a check after `launchAfterPermissionResult`